docs/integrations/dbt.md

```diff
@@ -22,36 +25,43 @@ The target specified in your `profiles.yml` file will be used by default. The ta
 
 ### Running SQLMesh
 
-Link to how to normally run sqlmesh here (plan, run). Continue to use your dbt format.
+You run SQLMesh as with any SQLMesh project, generating and applying [plans](../concepts/overview.md#make-a-plan), running [tests](../concepts/overview.md#tests) or [audits](../concepts/overview.md#audits), and executing models with a [scheduler](../guides/scheduling.md) if desired. 
```

Comment:
Should we use the word 'you; in our documentation? I thought that was frowned upon?

Reply:
I think using second-person voice is generally recommended, like in this [google style guide](https://developers.google.com/style/person). 

But the "you" is usually implicit (like in imperative statements "do X"), and I agree this sounds awkward!